docs/quickstart/regression.md

```diff
@@ -0,0 +1,122 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Regression
+
++++
```

Comment:
tip: the first 1-2 sentences should tell the reader what they will learn "in this tutorial, we'll use sklearn and sklearn-evaluation to fit and evaluate a regression model"